### PR TITLE
Variant Normalization Blog

### PR DESCRIPTION
**Pull request by kianfar77**
## What changes are proposed in this pull request?

This PR adds a blog page to the docs and puts the variant normalization blog as the first blog in it.

## How is this patch tested?
- [ ] Manual tests

### CODE REVIEW

File: docs/source/blogs/index.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Blog Posts
+==========
+
+.. toctree::
+   :maxdepth: 1
+
+   [Nov. 2019] Streamlining Variant Normalization on Large Genomic Datasets <variant-normalization-blog/variant-normalization-blog>
```
Comment (henrydavidge) [inline docs/source/blogs/index.rst]: Do we need to put this link explicitly? Typically we let the toctree directive list all the blogs.

File: docs/source/blogs/variant-normalization-blog/variant-normalization-blog.rst [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+To avoid incorrect bias in the results of downstream analysis on the merged set of variants or waste of analysis effort on seemingly new variants due to lack of normalization, which are in fact redundant (see Tan et al. for examples of this redundancy in 1000 Genome Project variant calls and dbSNP)
```
Comment (henrydavidge) [inline docs/source/blogs/variant-normalization-blog/variant-normalization-blog.rst]: Should this be a list?

File: docs/source/blogs/variant-normalization-blog/variant-normalization-blog.rst [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+To ensure that the merged data set and its post-analysis derivations are compatible and comparable with other public and private variant databases.
+
+This is achieved by what is referred to as variant normalization, a process that ensures the same variant is represented identically across different data sets. Performing variant normalization on terabytes of variant data in large projects using popular single-node tools can become quite a challenge as the acceptable input and output to these tools are the flat file formats that are commonly used to store variant calls (such as VCF and BGEN). To address this issue, we introduced the variant normalization transformation into Glow, which directly acts on a Spark Dataframe of variants to generate a DataFrame of normalized variants, harnessing the power of Spark to normalize variants from hundreds of thousands of samples in a fast and scalable manner with just a single line of Python or Scala code. Before addressing our normalizer, let us have a slightly more technical look at what variant normalization actually does.
```
Comment (henrydavidge) [inline docs/source/blogs/variant-normalization-blog/variant-normalization-blog.rst]: nit: to these tools -> of these tools

File: docs/source/blogs/index.rst (hunk 1 above)
Comment (kianfar77) [reply docs/source/blogs/index.rst]: This is to have the date of the blog in toc but not in the title line of the blog itself.

File: docs/source/blogs/variant-normalization-blog/variant-normalization-blog.rst (hunk 2 above)
Comment (kianfar77) [reply docs/source/blogs/variant-normalization-blog/variant-normalization-blog.rst]: Yes! lost in translation. fixed.

**Pull request merged**